app/templates/search/_services_summary.html

```diff
@@ -2,4 +2,20 @@
   <p class="search-summary">
     {{ summary }}
   </p>
-</div>
+
+  {% if total == 0 %}
+
+  <div class="search-suggestion">
```

Comment:
CSS updated in the toolkit: https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/360/files